base/init-job.yaml

```diff
@@ -8,58 +8,30 @@ spec:
   ttlSecondsAfterFinished: 600
   template:
     spec:
-      initContainers:
-      - name: init-certs
-        image: cockroach-cfssl-certs
-        imagePullPolicy: Always
-        command: ["cockroach-certs"]
-        env:
-        - name: CERTIFICATE_TYPE
-          value: "client"
-        - name: USER
-          value: "root"
-        - name: CERTS_DIR
-          value: "/cockroach-certs"
-        - name: CA_PROFILE
-          valueFrom:
-            configMapKeyRef:
-              name: ca.config
-              key: ca.client.profile
-        - name: CA_ADDRESS
-          valueFrom:
-            configMapKeyRef:
-              name: ca.config
-              key: ca.endpoint
-        - name: CA_AUTH_KEY
-          valueFrom:
-            secretKeyRef:
-              name: ca.auth
-              key: key
-        volumeMounts:
-        - name: client-certs
-          mountPath: /cockroach-certs
       containers:
-      - name: cluster-init
-        image: cockroachdb
-        imagePullPolicy: IfNotPresent
-        volumeMounts:
-        - name: client-certs
-          mountPath: /cockroach-certs
-        env:
-        - name: COCKROACH_INIT_HOST
-          value: cockroachdb-0.cockroachdb
-        command:
-        - "/bin/bash"
-        - "-c"
-        - "/cockroach/cockroach init --certs-dir=/cockroach-certs --host=$(COCKROACH_INIT_HOST) --port=26357 2>&1 | grep 'initialized'"
-        resources:
-          requests:
-            cpu: 0
-            memory: 128Mi
-          limits:
-            cpu: 1
-            memory: 512Mi
+        - name: cluster-init
+          image: cockroachdb
+          imagePullPolicy: IfNotPresent
+          volumeMounts:
+            - name: client-certs
+              mountPath: /cockroach/cockroach-certs
```

Comment:
Why the nesting?

Reply:
cockroach statefulset (server) had this path format. I thought it would be better to have init and client using the same format. just makes it cleaner in my eyes